Sources/ArcGISToolkit/Components/Offline/Preplanned/PreplannedMapModel.swift

```diff
@@ -22,7 +22,7 @@ internal import os
 @MainActor
 class PreplannedMapModel: ObservableObject, Identifiable {
     /// The preplanned map area.
-    let preplannedMapArea: any PreplannedMapAreaProtocol
+    let preplannedMapArea: PreplannedMapAreaProtocol
```

Comment:
I didn't realize that this PR was ready for re-review and was surprised to see that it has been merged. As part of [this](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/pull/1136#discussion_r2033768148) suggestion, the change on this line should have been reverted as well.

Reply:
Sorry for missing this. I'll fix it real quick